site/content/docs/5.3/customize/color.md

```diff
@@ -365,19 +365,36 @@ Be sure to monitor contrast ratios as you customize colors. As shown below, we'v
         {{ $color.hex }}
       </div>
       {{ range slice 50 | append (seq 100 100 900) }}
-      <div class="p-3 bd-{{ $color.name }}-{{ . }}">${{ $color.name }}-{{ . }}</div>
+      <div class="p-3 bd-{{ $color.name }}-{{ . }}">
+        <div class="d-flex flex-column">
+          <span>${{ $color.name }}-{{ . }}</span>
+          <div style="font-size: 0.75rem" class="d-flex flex-column">
```

Comment:
why is d-flex needed?

Reply:
I was using span, I can remove d-flex by changing span to div.